builder/test/softmax.cc

```diff
@@ -107,7 +107,7 @@ INSTANTIATE_TEST_SUITE_P(mode, softmax,
     test_params_to_string<softmax::ParamType>);
 
 INSTANTIATE_TEST_SUITE_P(compute_at, softmax,
-    testing::Combine(testing::Values(1), testing::Values(1), testing::Values(false), testing::Values(0)),
+    testing::Combine(testing::Values(1), testing::Values(1), testing::Values(true), testing::Values(0)),
```

Comment:
should this be `false, true`?

Reply:
false is covered by the test right above this one.

Reply:
Should that one be `false, true` instead? i.e. should we test split factors with compute_at?

Reply:
Sure, there is a small subset of parameters when it doesn't make sense to compute_at (b == 0), so it will be just equal to the test where compute_at is false though.

Reply:
That's fine, there's a lot of that in our tests already